.circleci/config.yml

```diff
@@ -187,14 +188,15 @@ jobs:
             docker_container_tag = "$DOCKER_CONTAINER_TAG"
             user_assigned_identity_name = "$AP_USER_ASSIGNED_ENTITY"
             END
-            terraform import -var-file="$TERRAFORM_ENV_CONFIG_FILE" azurerm_resource_group.rg /subscriptions/$AZ_SUBSCRIPTION_ID/resourceGroups/$RESOURCE_GROUP_NAME || true
-            terraform import -var-file="$TERRAFORM_ENV_CONFIG_FILE" azurerm_service_plan.sp /subscriptions/$AZ_SUBSCRIPTION_ID/resourceGroups/$RESOURCE_GROUP_NAME/providers/Microsoft.Web/serverfarms/$SERVICE_PLAN_NAME || true
+            terraform import -var-file="$TERRAFORM_ENV_CONFIG_FILE" azurerm_resource_group.app_plan_rg /subscriptions/$AZ_SUBSCRIPTION_ID/resourceGroups/$APP_PLAN_RESOURCE_GROUP_NAME || true
+            terraform import -var-file="$TERRAFORM_ENV_CONFIG_FILE" azurerm_service_plan.sp /subscriptions/$AZ_SUBSCRIPTION_ID/resourceGroups/$APP_PLAN_RESOURCE_GROUP_NAME/providers/Microsoft.Web/serverfarms/$SERVICE_PLAN_NAME || true
+            terraform import -var-file="$TERRAFORM_ENV_CONFIG_FILE" azurerm_resource_group.rg /subscriptions/$AZ_SUBSCRIPTION_ID/resourceGroups/$SERVICE_NAME$ENVIRONMENT_SUFFIX_LOWERCASE || true
```

Comment:
The best is to create a RESOURCE_GROUP_NAME env variable with $SERVICE_NAME$ENVIRONMENT_SUFFIX_LOWERCASE and reuse it. We shouldn't change all references, just the first ones, azurerm_resource_group.app_plan_rg & azurerm_service_plan.sp